pkg/volume/util/util.go

```diff
@@ -144,11 +150,27 @@ func PathExists(path string) (bool, error) {
 		return true, nil
 	} else if os.IsNotExist(err) {
 		return false, nil
+	} else if IsStaleConnection(err) {
+		return true, err
 	} else {
 		return false, err
 	}
 }
 
+func IsStaleConnection(err error) bool {
+	switch pe := err.(type) {
+	case nil:
+		return false
+	case *os.PathError:
```

Comment:
1. better to change to

```
case *os.PathError, *os.LinkError, *os.SyscallError
```

2. at least we should have a `default` for `switch`.

3. `err = pe.Err` ? Wrong? I don't think the `err` has anything to do with `type switch`.

Reply:
the code is copied from /usr/local/go/src/os/error.go#underlyingError
1. The change make compile errors
2. I will add one
3. I will change the var name to underlyingError